soarca_fin_python_library/executor.py

```diff
@@ -131,19 +131,19 @@ def _handle_register_message(self, message: Register):
                 self._wait_for_ack(message.message_id)
                 return
             # Did not get a response in time
-            except (Empty, TimeoutError) as e:
+            except (Empty, TimeoutError):
                 # Resend message
                 self._send_message_as_json(message, topic="soarca")
 
             # Receive a response but the message failed (Nack)
-            except RuntimeError as e:
```

Comment:
Maybe log the error where you removed the e?

Reply:
Now calls log.debug. In the wait_for_ack method log.info and log.warning are called to provide some additonal context.